kyaml/fn/runtime/exec/exec.go

```diff
@@ -28,6 +32,15 @@ func (c *Filter) Filter(nodes []*yaml.RNode) ([]*yaml.RNode, error) {
 }
 
 func (c *Filter) Run(reader io.Reader, writer io.Writer) error {
+	if c.WorkingDir != "" {
+		p, err := os.Getwd()
+		if err != nil {
+			return err
+		}
+		os.Chdir(c.WorkingDir)
```

Comment:
Doesn't this return an error?

Reply:
Actually, `exec.Cmd` has a `Dir` field that controls the working dir for you. Can we use that?

```go
	// Dir specifies the working directory of the command.
	// If Dir is the empty string, Run runs the command in the
	// calling process's current directory.
	Dir string
```

Reply:
Thanks for pointing that out! I've changed it to use that field rather than using os.Chdir.